src/lib/Preview.js

```diff
@@ -814,6 +814,11 @@ class Preview extends EventEmitter {
      * @return {void}
      */
     setupUI() {
+        // Do nothing if container is already setup and in the middle of retrying
+        if (this.container && this.retryCount > 0) {
```

Comment:
would this.open be helpful here?

Reply:
I'm not sure, it seems like `this.open` is set when `load` is first called and is no guarantee of the preview UI having been setup already